blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx

```diff
@@ -62,7 +62,8 @@ export class RunDetailsPipeline extends Component {
             }
         }
 
-        this.listener.sse = sseConnection.subscribe('pipeline', this._onSseEvent);
+        this.listener.ssePipeline = sseConnection.subscribe('pipeline', this._onSseEvent);
+        this.listener.sseJob = sseConnection.subscribe('job', this._onSseEvent);
```

Comment:
Hmmm ... one listener receiving events from different channels? Yes it can work that way, but not sure I like that idea tbh ... seems like a recipe for confusing other people that will be reading the code => possible source of those people making bad changes to the code ... bugs etc